src/TextureManager.h

```diff
@@ -13,6 +13,7 @@
 #include <string>
 #include <map>
 #include "SDL2/SDL.h"
+#include "Utils/Printer.h"
```

Comment:
Same problem as before, this is on utils

Reply:
mmm, no se que te sale en el repo cuando pulleaste, pero a mi me sale que existe solo Utils en mayuscula, lo modifiqué en el CMakeLists para que quede con el mayúscula

Reply:
![image](https://user-images.githubusercontent.com/28717840/97767882-fa1a2e80-1afd-11eb-8393-df7578b6fcc7.png)